CMakeLists.txt

```diff
@@ -90,6 +90,20 @@ if(CERES_FOUND)
   include_directories(${CERES_INCLUDE_DIRS})
 endif(CERES_FOUND)
 
+find_package(HDF5 1.8 COMPONENTS CXX)
+
+if(HDF5_FOUND)
+  message("-- HDF5 found for checkpointing")
+  message("   Version " + ${HDF5_VERSION})
+  include_directories(${HDF5_INCLUDE_DIRS})
+
+  if (${HDF5_VERSION} GREATER 1.8)
+    message(WARNING "HDF5 will be used such that it is compatible with version 1.8.")
+  endif()
+
+endif()
```

Comment:
What is happening if `HDF5` isn't found?

Reply:
I think HDF5 will be required as we are getting rid of the old orb files. So CMake should fail if there isn't any HDF5.

Reply:
Ok, so you add `REQUIRED` above.

Reply:
Okay, done. Thanks!